Xamarin.Forms.Platform.iOS/CollectionView/ObservableItemsSource.cs

```diff
@@ -10,17 +10,21 @@ internal class ObservableItemsSource : IItemsViewSource
 	{
 		readonly UICollectionView _collectionView;
 		readonly IList _itemsSource;
+		bool _isEmpty;
+
 
 		public ObservableItemsSource(IEnumerable itemSource, UICollectionView collectionView)
 		{
 			_collectionView = collectionView;
 			_itemsSource = (IList)itemSource;
```

Comment:
Is this safe? I just noticed that we just cast. It might be better to change the constructor parameter to `IList` - this way we move the responsibility to the caller.